src/rust/src/x509/verify.rs

```diff
@@ -287,6 +330,7 @@ fn build_subject<'a>(
 
             Ok(Subject::IP(ip_addr))
         }
+        SubjectOwner::None => Err(pyo3::exceptions::PyValueError::new_err("missing subject")),
```

Comment:
Flagging: this is kind of nasty -- we need it for exhaustion reasons, but it should be unreachable in practice (since this variant is never passed into `build_subject`).